sources/ingest/importers/utils/tests/test_opening_hours.py

```diff
@@ -80,7 +82,67 @@
                             "periods": [1306],
                         }
                     ],
-                }
+                },
+                {
+                    "date": "2021-09-03",
+                    "times": [
+                        {
+                            "name": "",
+                            "description": "",
+                            "start_time": "10:00:00",
+                            "end_time": "11:00:00",
+                            "end_time_on_next_day": False,
+                            "resource_state": "closed",
+                            "full_day": False,
+                            "periods": [1306],
+                        }
+                    ],
+                },
+                {
+                    "date": "2021-09-04",
+                    "times": [
+                        {
+                            "name": "",
+                            "description": "",
+                            "start_time": None,
+                            "end_time": None,
+                            "end_time_on_next_day": False,
+                            "resource_state": "open",
+                            "full_day": True,
+                            "periods": [1306],
+                        }
+                    ],
+                },
+                {
+                    "date": "2021-09-05",
+                    "times": [
+                        {
+                            "name": "",
+                            "description": "",
+                            "start_time": None,
+                            "end_time": None,
+                            "end_time_on_next_day": False,
+                            "resource_state": "closed",
+                            "full_day": True,
+                            "periods": [1306],
+                        }
+                    ],
+                },
+                {
+                    "date": "2021-09-06",
+                    "times": [
+                        {
+                            "name": "",
+                            "description": "",
+                            "start_time": "09:00:00",
+                            "end_time": "05:00:00",
+                            "end_time_on_next_day": True,
+                            "resource_state": "open",
+                            "full_day": False,
+                            "periods": [1306],
```

Comment:
It's hard to see the differences here and if every necessary field combinations are tested. Could every entry base their data on some default dict, so it would be easier to see the differences in objects? If the data is some real life data, I think it should be imported from a fixture.

Reply:
Yeah true. However, I don't see any trivial fixes, and I don't think that is a major problem. If more opening hours tests are needed in the future, this is something that should be considered.